src/components/__tests__/Menu.js

```diff
@@ -313,27 +313,27 @@ describe('default render', () => {
 
     expect(toggleShowMore).toHaveBeenCalledTimes(1);
   });
+});
 
-  it('calls the Panel mixin with `canRefine`', () => {
-    __setState({ ...defaultState });
-
-    const wrapper = mount(Menu, {
-      propsData: defaultProps,
-    });
+it('calls the Panel mixin with `canRefine`', () => {
+  __setState({ ...defaultState });
 
-    const mapStateToCanRefine = () =>
-      wrapper.vm.mapStateToCanRefine(wrapper.vm.state);
+  const wrapper = mount(Menu, {
+    propsData: defaultProps,
+  });
 
-    expect(mapStateToCanRefine()).toBe(true);
+  const mapStateToCanRefine = () =>
+    wrapper.vm.mapStateToCanRefine(wrapper.vm.state);
 
-    wrapper.setData({
-      state: {
-        canRefine: false,
-      },
-    });
+  expect(mapStateToCanRefine()).toBe(true);
 
-    expect(mapStateToCanRefine()).toBe(false);
+  wrapper.setData({
+    state: {
+      canRefine: false,
+    },
   });
+
```

Comment:
```suggestion
     expect(mapStateToCanRefine()).toBe(false); 
```

I guess this was deleted by accident?

Reply:
yep, thanks!